pilot/pkg/networking/core/v1alpha3/envoyfilter/rc_patch.go

```diff
@@ -46,32 +68,34 @@ func ApplyRouteConfigurationPatches(
 
 	applied := false
 
+	portMap := getTargetPortMap(proxy.MergedGateway)
```

Comment:
Can this port map be part of MergedGateway so that we do not recompute on every patch apply?

Reply:
Good idea